doc/api/cli.md

```diff
@@ -111,6 +111,10 @@ Process v8 profiler output generated using the v8 option `--prof`.
 
 Print v8 command line options.
 
+Note: v8 options allow words to be separated by both dashes (`-`) or underscores
+(`_`).
+
+For example, `--stack-trace-limit` is equivalent to `--stack_trace_limit`.
```

Comment:
nit: it's likely not worth noting but `--stack-trace_limit` and `--stack_trace-limit` are also equivalent.